app/javascript/application.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import { store } from "./redux/configureStore";
+import { Provider } from "react-redux";
+
+import Greeting from "./components/Greeting.js";
+
+function App() {
+  return (
+    <div>
+      <Greeting />
+    </div>
+  );
```

Comment:
- [ ] Routes should be setup using react-router-dom and the App component should have a route leading to the greeting component.

Reply:
Hi @mikethreels, thanks for the feedback. All required changes have been implemented.